libfwupdplugin/fu-config.c

```diff
@@ -165,9 +165,16 @@ fu_config_reload(FuConfig *self, GError **error)
 		g_autofree gchar *dirname = g_path_get_dirname(item->filename);
 		g_debug("trying to load config values from %s", item->filename);
 		if (g_file_query_exists(item->file, NULL)) {
-			g_autoptr(GBytes) blob = fu_bytes_get_contents(item->filename, error);
-			if (blob == NULL)
+			g_autoptr(GError) error_load = NULL;
+			g_autoptr(GBytes) blob = fu_bytes_get_contents(item->filename, &error_load);
+			if (blob == NULL) {
+				if (g_error_matches(error_load, G_FILE_ERROR, G_FILE_ERROR_ACCES)) {
+					g_debug("%s", error_load->message);
+					continue;
+				}
+				g_propagate_error(error, g_steal_pointer(&error_load));
 				return FALSE;
+			}
 			fu_byte_array_append_bytes(buf, blob);
 		}
```

Comment:
maybe we drop the g_file_query_exists() and handle the extra failure case?

Reply:
changed